src/xcode/ENA/ENA/Source/Services/PPAnalyticsCollector/PPAnalyticsCollector.swift

```diff
@@ -284,11 +285,16 @@ enum PPAnalyticsCollector {
 		case let .submittedAfterSymptomFlow(afterSymptomFlow):
 			store?.keySubmissionMetadata?.submittedAfterSymptomFlow = afterSymptomFlow
 		case let .submittedWithTeletan(withTeletan):
```

Comment:
i do not like negations. Why not following:

```suggestion
		case let .submittedWithQr(withQr):
		store?.submittedWithQR = withQr
```

Reply:
I would rather prefer submittedWithTeletan to align with what we need to send to the api and what we have in the store i.e., submittedWithTan.